xc/common/utils/prjxray_arch_import.py

```diff
@@ -526,12 +534,22 @@ def make_prefix(site, x, y, from_site_name=False):
     prefix_required = PREFIX_REQUIRED[site_type]
 
     if prefix_required[0] == 'Y':
-        return site_type, '{}_Y{}'.format(site_type, y % prefix_required[1])
+        mod_y = prefix_required[1]
+        y_formula = "y{}".format(" % mod_y") if mod_y else "y"
+        return site_type, '{}_Y{}'.format(site_type, eval(y_formula))
```

Comment:
Please do not use eval.  I don't believe it is required here.

Reply:
Yep, removed eval